src/BUILD.gn

```diff
@@ -73,8 +73,8 @@ target("executable", "jsitests") {
     "//testing/gtest",
   ] 
 
-  configs += [ "//:internal_config_base", "//build/config/compiler:exceptions" ]
-  configs -= [ "//build/config/compiler:no_exceptions" ]
+  configs += [ "//:internal_config_base", "//build/config/compiler:exceptions", "//build/config/compiler:rtti" ]

```

Comment:
Does enabling rtti has any impact on the binary size ? It is interesting from Andooid platform POV ..

Reply:
This is only for the test executable, shouldn't impact the actual v8jsi library.